samples/multi-agent/src/main/java/demo/multiagent/application/agents/WeatherAgent.java

```diff
@@ -67,7 +67,7 @@ private String getWeather(
   }
   // end::function-tool[]
 
-  @FunctionTool(description = "Return current date in yyyy-MM-dd format")
+  @FunctionTool("Return current date in yyyy-MM-dd format")
```

Comment:
That's quite some stuff for this reduction, which I'm not even sure if it reads better.
What happens when adding `name`? Would that be
```
@FunctionTool("Return current date in yyyy-MM-dd format", name = "current_date")
```

Reply:
then you have to write it as:

```java
@FunctionTool(value = "Return current date in yyyy-MM-dd format", name = "current_date")
```

Assuming that `FunctionTool` is defined as: 

```java
@Retention(RetentionPolicy.RUNTIME)
@Documented
public @interface FunctionTool {

  String name() default "";
  String value();

}
```